src/entities/communitySbt.ts

```diff
@@ -428,6 +450,85 @@ class SBT {
         return badgeResponseRecord;
     }
 
+    public async getReferrorBadges(userId: string, referroorBadgesUrl: string, subgraphUrl: string, seasonId: number) : Promise<Record<string, BadgeResponse>> {
+        let badgeResponseRecord : Record<string, BadgeResponse> = {};
+
+        const resp = await axios.get(`${referroorBadgesUrl}/referrals-by/${userId.toLowerCase()}`);
+        if (!resp.data){
+            return badgeResponseRecord;
+        }
+
+        const referees: string[] = resp.data;
+        let refrees100K = 0;
+        let refrees2M = 0;
+        for (const referee of referees){
+            const badgeQuery = `
+                query( $id: String) {
+                    seasonUsers( where: {owner_contains: $id}) {
+                        id
+                        totalWeightedNotionalTraded
+                      
+                    }
+                }
+            `;
+            const client = new ApolloClient({
+                cache: new InMemoryCache(),
+                link: new HttpLink({ uri: subgraphUrl, fetch })
+            })
+            const id = `${referee.toLowerCase()}`
+            const data = await client.query<{
+                seasonUsers: {
+                    totalWeightedNotionalTraded: string
+                }[]
+            }>({
+                query: gql(badgeQuery),
+                variables: {
+                    id: id,
+                },
+            });
+
+            if(!data?.data?.seasonUsers){
+                continue;
+            }
+
+            let totalPointz = BigNumber.from(0);
+            data.data.seasonUsers.forEach((user) => {
+                totalPointz = totalPointz.add(
+                    BigNumber.from(user.totalWeightedNotionalTraded)
+                );
+            });
+            if(totalPointz.gte(BigNumber.from("100000"))) {
```

Comment:
extract 100000 and similar constants to a constant file or move them to the top of file giving them proper naming so that it is easy for us to change them in future.